hafnian/samples.py

```diff
@@ -468,7 +491,7 @@ def torontonian_sample_graph(A, n_mean, samples=1, max_photons=30, pool=False):
     return torontonian_sample_state(cov, samples, hbar=2, max_photons=max_photons, pool=pool)
 
 
-def hafnian_sample_classical_state(cov, samples, mean=None, hbar=2, atol=1e-08):
+def hafnian_sample_classical_state(cov, samples, mean=None, hbar=2, atol=1e-08, cutoff=None):
```

Comment:
How come `cutoff` was added here, but isn't being used anywhere within the function?

Reply:
It is just to keep consistency with the other functions. Eventually, we can add the possibility of removing samples with more than cutoff photons in a given mode. The reason I added this argument was so that I could pytest parametrize in the test with the same args as hafnian_classical_state

Reply:
Ah okay! Best to leave a comment for the linter:
```suggestion
def hafnian_sample_classical_state(cov, samples, mean=None, hbar=2, atol=1e-08, cutoff=None): # add cutoff for consistency pylint: disable=unused-argument
```